docs/GettingStarted.md

```diff
@@ -2122,26 +2122,8 @@ By default, this will be activated whenever `ddtrace` detects the application is
 
 ### Sampling
 
-#### Application-side sampling
-
-While the trace agent can sample traces to reduce bandwidth usage, application-side sampling reduces the performance overhead.
-
-The default sampling rate can be set between `0.0` (0%) and `1.0` (100%). Configure the rate in order to control the volume of traces sent to Datadog. When this configuration is not set, the Datadog agent will distribute a default sampling rate of 10 traces per second.
-
-Set this value via `DD_TRACE_SAMPLE_RATE` or `Datadog.configure { |c| c.tracing.sampling.default_rate = <value> }`.
```

Comment:
I'm curious -- what's the difference between the two approaches presented and why omit this variant from the updated docs?

Reply:
This part of documentation kind of mixes up application-side sampling with agent-based sampling: "`DD_TRACE_SAMPLE_RATE` or `Datadog.configure { |c| c.tracing.sampling.default_rate = <value> }`" are agent-based, despite being in this section. I think it was just kind of confusing before.

But in general, environment variable (`DD_TRACE_SAMPLE_RATE`) vs `Datadog.configure { |c| c.tracing.sampling.default_rate = <value> }` are the same thing, but we'd rather have clients using environment variables if possible, as programmatic configuration might not be realist in a world where we support YAML and remote configuration.

Normally, across other Datadog products, we try to recommend environment variables, if they have been standardized, over programmatic approaches.
I normally only list the programmatic approach if it helps people find such configuration in their current codebase, in case we are advertising that changes are needed in that specific part of their codebase.